### ISSUE TITLE
command "init --remote" fails with message "Failed to fetch user data"

### ISSUE THREAD

**Issue by seyaobey-dev** [nhost/cli#132]
**Describe the bug**
I'm using `nhost v2`. I have created a remote repository and now I'm trying to spin up a local repository and link it with the remote. When I run `nhost init --remote`, I get `Failed to fetch user data` when I supply my email and password (correct values).

**To Reproduce**
1. Create a remote repository
2. Run `nhost init --remote`

**Expected behavior**
Process complete smoothly

**Screenshots**
![Screenshot at Dec 21 18-36-33](https://user-images.githubusercontent.com/83374925/146981310-4b66560b-e124-460a-92e8-016ab4700017.png)

**Desktop (please complete the following information):**
 - OS: macos monterey
 - Version: `Nhost CLI v0.5.11 for darwin-arm64`

**Comment by spkprav** [comment]
Same here, I get `Failed to pull migrations from remote` instead.

**Comment by mrinalwahal** [comment]
Can you please run `nhost init --remote --debug` and confirm what is the exact point at which the CLI is throwing these errors? The debug flag should show you a log trace.

**Comment by artistic-differences** [comment]
Not working for me either. I have created an account using the github login integration.
Despite this when I try to login using: `nhost login` (or any method that requires auth)
The command asks me for my email and password (account does not have a password)

**Comment by mrinalwahal** [comment]
> Not working for me either. I have created an account using the github login integration. Despite this when I try to login using: `nhost login` (or any method that requires auth) The command asks me for my email and password (account does not have a password)

You can save your password on Nhost dashboard. And then use that one to login.

**Comment by mrinalwahal** [comment]
Since, there aren't any responses and this does seems potentially a problem from user's end, I'm closing this issue.

We can open it again if anyone can't find their way around it.

**Comment by sangdth** [comment]
Sometimes I got this as well. Do we have a wait to bring you log file or sth similar?

**Comment by artistic-differences** [comment]
Hi Sang,
Thanks for the prompt response. There is indeed the option to save a password against my account and I tried that several times last night. First whilst logged in and then after logout and login. On each occasion there was an invalid access token error.
Thanks, Ross  
    On Tuesday, 11 January 2022, 06:44:48 GMT, Sang Dang ***@***.***> wrote:  
 
 


Sometimes I got this as well. Do we have a wait to bring you log file or sth similar?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
